adapter/src/main/java/org/jvalue/ods/adapterservice/datasource/api/rest/v1/DatasourceEndpoint.java

```diff
@@ -88,10 +88,8 @@ public void deleteAllDatasources() {
       .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND, "Can not find Datasource with id: " + id));
     try {
       return datasourceManager.trigger(id, runtimeParameters);
-    } catch (IllegalArgumentException e) {
-      throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Invalid datasource or parameters: " + e.getMessage());
-    } catch (RestClientException e) {
-      throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Failed to load data: " + e.getMessage());
+    } catch (ResponseStatusException e) {
```

Comment:
unnecessary

Reply:
This is actually needed to prevent all response status exceptions from being caught by the subsequent catch clause.

Reply:
Ok thanks, didn't see that!